bitrepository-client/src/main/java/org/bitrepository/commandline/PutFileCmd.java

```diff
@@ -108,16 +108,17 @@ protected void createOptionsForCmdArgumentHandler() {
         cmdHandler.addOption(checksumOption);
 
         Option checksumTypeOption = new Option(Constants.REQUEST_CHECKSUM_TYPE_ARG, Constants.HAS_ARGUMENT,
-                "[OPTIONAL] Used to request a specific checksum algorithm in the response from the pillars.");
+                Constants.REQUEST_CHECKSUM_TYPE_DESC);
         checksumTypeOption.setRequired(ARGUMENT_IS_NOT_REQUIRED);
         cmdHandler.addOption(checksumTypeOption);
+
         Option checksumSaltOption = new Option(Constants.REQUEST_CHECKSUM_SALT_ARG, Constants.HAS_ARGUMENT,
-                "[OPTIONAL] Used to request a salted checksum in the response. Requires the ChecksumType argument.");
+                Constants.REQUEST_CHECKSUM_SALT_DESC);
         checksumSaltOption.setRequired(ARGUMENT_IS_NOT_REQUIRED);
         cmdHandler.addOption(checksumSaltOption);
 
         Option deleteOption = new Option(Constants.DELETE_FILE_ARG, Constants.NO_ARGUMENT,
-                "If this argument is present, then the file will be removed from the server, when the chosen operation is complete.");
```

Comment:
Nitpicking now we're at it: remove the second comma, the one between "server" and "when".